### PR TITLE
Used the new fluent interface for DefinitionBuilder example

### LINKED ISSUE #7693: [Workflow] Added fluent interface to the DefinitionBuilder
See https://github.com/symfony/symfony/pull/21950

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7693.

### CODE REVIEW

File: components/workflow.rst [hunk 1]
```diff
@@ -44,19 +44,23 @@ these statuses are called **places**. You can define the workflow like this::
...
+        ->addTransition(new Transition('publish', 'review', 'published'))
+        ->addTransition(new Transition('reject', 'review', 'rejected'))
+        ->build()
+    ;
 
     $marking = new SingleStateMarkingStore('currentState');
     $workflow = new Workflow($definition, $marking);
 
+.. versionadded:: 3.3
+    The fluent interface for ``DefinitionBuilder`` class was added in Smyfony 3.3.
```
Comment (xabbuh) [inline components/workflow.rst]: [...] for the `DefinitionBuilder` [...]

Comment (xabbuh) [inline components/workflow.rst]: [...] was introduced [...]

File: components/workflow.rst [hunk 2]
```diff
@@ -44,19 +44,23 @@ these statuses are called **places**. You can define the workflow like this::
...
+    Before you had to call ``addPlaces()``, ``addTransition()`` and ``build()``
```
Comment (xabbuh) [inline components/workflow.rst]: [...] had to call the [...]

**Comment by javiereguiluz** [comment]
I don't think anyone can make more mistakes in such a short paragraph! Luckily we have your reviews ... so I've fixed everything. Thanks!

**Review by xabbuh — APPROVED** [review]
:+1:

**Review by HeahDude — APPROVED** [review]

File: components/workflow.rst (hunk 2 above)
Comment (ming820214) [reply components/workflow.rst]: but,the example occur "Fatal error: Class 'Symfony\Component\Workflow\DefinitionBuilder' not found ",i use the symfony 3.3.10 version,why?

File: components/workflow.rst (hunk 1 above)
Comment (ming820214) [reply components/workflow.rst]: but,the example occur "Fatal error: Class 'Symfony\Component\Workflow\DefinitionBuilder' not found ",i use the symfony 3.3.10 version,why?

File: components/workflow.rst (hunk 2 above)
Comment (xabbuh) [reply components/workflow.rst]: The class is there: https://github.com/symfony/symfony/blob/v3.3.10/src/Symfony/Component/Workflow/DefinitionBuilder.php

Are you sure that you use Symfony 3.3.10? I suggest that you use one of [the support channels](https://symfony.com/support) to seek help with your problem.

**Pull request closed without merging**